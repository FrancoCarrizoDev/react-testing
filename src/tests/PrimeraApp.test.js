import React from 'react';

import PrimeraApp from "../PrimeraApp"
import { shallow } from 'enzyme'

describe('Test in PrimeraApp', () => {
    /*
    test('should show an message "Hellow im goky"', () => {
        const greeting = "Hi! i'm Goku"

        const {getByText} = render(<PrimeraApp saludo={greeting}/>)
        expect( getByText(greeting)).toBeInTheDocument();
    })
    */

    test('should show <PrimeraApp/> correctly', () => {
        const wrapper = shallow(<PrimeraApp saludo="Hellow!" />)

        expect(wrapper).toMatchSnapshot();
    })

    test('should show the subtitle sneded by props', () => {
        const greeting = 'Hello im Goku!'
        const subtitle = 'Im subtitle'
        const wrapper = shallow(<PrimeraApp saludo={greeting} subtitulo={subtitle} />)

        const parragrapthText = wrapper.find('p').text()

    })
    

})
