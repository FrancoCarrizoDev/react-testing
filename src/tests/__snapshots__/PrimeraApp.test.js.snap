// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in PrimeraApp should show <PrimeraApp/> correctly 1`] = `
<Fragment>
  <h1>
     
    Hellow!
     
  </h1>
  <p>
     
    Soy un subtitulo
     
  </p>
</Fragment>
`;
